src/Arcus.Messaging.Tests.Contracts/Events/v1/OrderCreatedEventData.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Arcus.Messaging.Abstractions;
+
+namespace Arcus.Messaging.Tests.Contracts.Events.v1
+{
+    public class OrderCreatedEventData
+    {
+        public OrderCreatedEventData(string id, int amount, string articleNumber, string customerName, MessageCorrelationInfo correlationInfo)
```

Comment:
Could be useful to add guard checks, some pre-assert phase to validate events?

Reply:
It's just for testing so it should be fine.

Don't want to add guards et al and further tweaks for serialization while it doesn't really matter here, no?

Reply:
Yeah I get it. Np